Branding/HomePage.html

```diff
@@ -24,6 +24,7 @@
 	margin: 0;
```

Comment:
The <kbd>Purchase Support</kbd> Link is redirecting to https://store.citrix.com/dr_product/citrix-hypervisor-subscription/ instead of https://store.citrix.com/dr_product/xenserver-subscription. Not 100% necessary for us to change it here, but I think it'd be nice.

Reply:
We may need a completely new link for this. Will get back to you once I find out more.